generators/entity/templates/src/main/webapp/app/entities/infinite-scroll-template.ejs

```diff
@@ -29,7 +29,11 @@
         <%_ } else { _%>
             if (vm.currentSearch) {
```

Comment:
Now that I look over the controllers for other pagination types I think we should make them more consistent. Maybe `currentSearch` can be removed now that pagingParams exists - testing will need to be done.